Makefile

```diff
@@ -87,8 +87,8 @@ uninstall: manifests kustomize ## Uninstall CRDs from a cluster
 	$(KUSTOMIZE) build config/crd | kubectl delete -f -
 
 .PHONY: deploy
-deploy: run-kind ## Deploy controller in the configured Kubernetes cluster in ~/.kube/config
-	@ act -j deploy -s KUBE_CONFIG_DATA='$(shell kind get kubeconfig | yq eval -j )'
```

Comment:
This is the same line I changed with my yq V4 commit so maybe that's the reason it started failing.
`yq eval` won't work with yq version 3 which in turn will spoil act -j command.

In any case now it's better =)